web/src/beta/components/DragAndDropList/Item.tsx

```diff
@@ -15,6 +16,7 @@ type Props = {
   itemGroupKey: string;
   id: string;
   index: number;
+  item?: any;
```

Comment:
Ah sorry, I guess you are only checking if the item has an extensionId, right? That seems like a lot of work to know if it is a StoryBlock or not. 
In actuality though, what you want to know is whether to make the WHOLE child the handle, or just the single element (in the StoryBlock's case, the dnd handle Icon, right?), right?

Reply:
If I won't check if is story telling the logic implemented here uses the ref which is used to drag and drop, I changed some of codes but I think we need to discuss so that I can explain why I did that